Example/Source/Examples/Collections/SimpleCollectionBrickViewController.swift

```diff
@@ -39,7 +39,7 @@ class SimpleCollectionBrickViewController: BrickApp.BaseBrickController {
         ])
 
         let section = BrickSection(bricks: [
-            CollectionBrick(backgroundColor: .brickSection, dataSource: CollectionBrickCellModel(section: collectionSection, configureHandler: { (brickCollectionCell) in
+            CollectionBrick(backgroundColor: .brickSection, dataSource: CollectionBrickCellModel(section: collectionSection, registerBricksHandler: { (brickCollectionCell) in
                 brickCollectionCell.brickCollectionView.registerBrickClass(LabelBrick.self)
```

Comment:
As you are registering just the class, shouldn't you add these brickTypes in the initializer of the CollectionBrick?